dynawo/sources/Models/Modelica/Dynawo/Electrical/HVDC/HvdcVSC/BaseControls/ActivePowerControl/ActivePowerControlDangling.mo

```diff
@@ -0,0 +1,93 @@
+within Dynawo.Electrical.HVDC.HvdcVSC.BaseControls.ActivePowerControl;
+
+/*
+* Copyright (c) 2015-2021, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+model ActivePowerControlDangling "Active power control for the HVDC VSC model that connects two different synchronous areas (P control on the side of the main one)"
```

Comment:
I prefer the wording you used in the HVDCPVDangling model for the comment. The problem is not the fact that the link connects two different synchronous areas but it is rather than one of them is not calculated.